bib/lsstdesc.bib

```diff
@@ -1,8 +1,8 @@
 % lsstdesc.bib
 % !! DO NOT EDIT! !!
```

Comment:
Do we want to take the opportunity to change this to "DO NOT EDIT (except by pull request)"?

Reply:
Heh. Yes, while we await the bright new automated future, this makes sense.